lib/i2cscan/i2cscan.cpp

```diff
@@ -23,47 +21,79 @@ uint8_t portExclude[] = {LED_PIN};
 
 namespace I2CSCAN
 {
+    enum ScanState {
+        IDLE,
+        SCANNING,
+        DONE
+    };
+
+    ScanState scanState = IDLE;
+    uint8_t currentI = 0;
+    uint8_t currentJ = 0;
```

Comment:
Would probably be best to give these meaningful names, like "currentSDA/currentSCL" or something